cbrain_task/fsl_melodic/views/_show_params.html.erb

```diff
@@ -29,17 +29,21 @@
 </p>
 
 <p>
-<STRONG>Functional file:</STRONG>
+<STRONG>Functional file(s):</STRONG>
 <ul>
-  <li><%= link_to_userfile_if_accessible(params[:functional_file_id]) %></li>
+  <% params[:functional_file_ids].each do |id| %>
+  <li><%= link_to_userfile_if_accessible(id) %></li>
+  <% end %> 
 </ul>
 </p>
 
-<% if !params[:structural_file_id].blank? %>
+<% if !params[:structural_file_ids].blank? %>
 <p>
-<STRONG>Structural file:</STRONG>
+<STRONG>Structural file(s):</STRONG>
 <ul>
-  <li><%= link_to_userfile_if_accessible(params[:structural_file_id]) %></li>
+  <% params[:structural_file_ids].each do |id| %>
+  <li><%= link_to_userfile_if_accessible(id) %></li>
+  <% end %>
 </ul>
 </p>
 <% end %>
```

Comment:
Lot of time I had this line : 

See this task's <%= link_to "edit page", edit_task_path(task) %> for more information.

in this way user have a link in order to see all the params.